lib/pre-build.js

```diff
@@ -46,21 +45,18 @@ preBuild.load = function load(target) {
   return require(path.join(BUILD_PATH, getBinFileName(target)))
 }
 
-preBuild.makePath = async function makePath(pathToMake) {
+preBuild.makePath = async function makePath() {
   const accessRights = fs.constants.R_OK | fs.constants.W_OK
 
-  // We only want to make the parts after the package directory.
-  pathToMake = path.join(PACKAGE_ROOT, pathToMake)
-
   try {
-    await fs.access(pathToMake, accessRights)
+    await fs.access(BUILD_PATH, accessRights)
   } catch (err) {
     if (err?.code !== 'ENOENT') {
       // It exists but we don't have read+write access! This is a problem.
-      throw new Error(`Do not have access to '${pathToMake}': ${err}`)
+      throw new Error(`Do not have access to '${BUILD_PATH}': ${err}`)
     }
 
-    await fs.mkdir(pathToMake, { recursive: true })
+    await fs.mkdir(BUILD_PATH, { recursive: true })
```

Comment:
so this was making a path of `<home>/node-native-metrics/<home>/node-native-metrics/build/Release` and it tries to save in `<home>/node-native-metircs/build/Release`. since that path did not exist it failed. So by removing `/build/Release` from the download server the path never exists as before it existed even before this function was called